### ISSUE TITLE
[BUG] Debian repo and pkgrepo_keyring partially correct

### ISSUE THREAD

**Issue by Wismerhilll** [saltstack-formulas/salt-formula#516]
When pinning the install to a minor release, (https://repo.saltproject.io/#debian) the repo and key_url addresses change from 
"pkgrepo: 'deb [signed-by=/usr/share/keyrings/salt-archive-keyring.gpg arch=amd64] {{ salt_repo }}/{{ py_ver_repr or 'apt' }}/{{ osfamily_lower }}/{{ osmajorrelease }}/amd64/{{ salt_release }} {{ oscodename }} main'
  pkgrepo_keyring: '{{ salt_repo }}/{{ py_ver_repr or 'apt' }}/{{ osfamily_lower }}/{{ osmajorrelease }}/amd64/{{ salt_release }}/salt-archive-keyring.gpg'
to
"pkgrepo: 'deb [signed-by=/usr/share/keyrings/salt-archive-keyring.gpg arch=amd64] {{ salt_repo }}/{{ py_ver_repr or 'apt' }}/{{ osfamily_lower }}/{{ osmajorrelease }}/amd64/archive/{{ salt_release }} {{ oscodename }} main'
  pkgrepo_keyring: '{{ salt_repo }}/{{ py_ver_repr or 'apt' }}/{{ osfamily_lower }}/{{ osmajorrelease }}/amd64/archive/{{ salt_release }}/salt-archive-keyring.gpg'"

Basically, if the release is "3003" it works, if the release is "latest" it works too, but id the release is 3003.3, it doesn't work.

Since I barely understand jinja, i won't try to fix it, but i think it should happen in the osfamilymap.yaml
